Src/IronPythonTest/Cases/CaseExecuter.cs

```diff
@@ -90,6 +90,30 @@ class CaseExecuter {
         }
 
         public int RunTest(TestInfo testcase) {
+            int retryCount = testcase.Options.RetryCount;
+            if(retryCount > 0) {
+                int res = -1;
+                for(int i = 0; i < retryCount; i++) {
+                    try {
+                        res = RunTestImpl(testcase);
```

Comment:
Is the loop break on success missing? Maybe I'm just tired.

Reply:
Haha, nope. Good catch!

Reply:
I meant, yes, it is missing.